index.html

```diff
@@ -1,49 +1,242 @@
+              <th>State</th>
+              <th>2014 Rate per 10K</th>
+              <th>Rate by Comparison</th>
+            </tr>
+          </thead>
+          <tbody>
+          </tbody>
+        </table>
+      </div>
     </div>
-    <table class="table">
-      <thead>
-        <tr>
-          <th>Fruit</th>
-          <th>Price</th>
-          <th>Quantity</th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr>
-          <td>Apples</td>
-          <td>$0.55</td>
-          <td>5</td>
-        </tr>
-        <tr>
-          <td>Oranges</td>
-          <td>$0.80</td>
-          <td>10</td>
-        </tr>
-        <tr>
-          <td>Pears</td>
-          <td>$0.95</td>
-          <td>3</td>
-        </tr>
-      </tbody>
-    </table>
-  </div>
-  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.17/d3.min.js"></script>
+
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.17/d3.min.js"></script>
+
+    <script>
+      var states = [
+        {"state":"AK","rate":24.3},
+        {"state":"AL","rate":9.4},
+        {"state":"AR","rate":9.9},
+        {"state":"AZ","rate":15.8},
+        {"state":"CA","rate":29.7},
+        {"state":"CO","rate":19.0},
+        {"state":"CT","rate":12.4},
+        {"state":"DC","rate":120.0},
+        {"state":"DE","rate":9.7},
+        {"state":"FL","rate":21.2},
+        {"state":"GA","rate":16.5},
+        {"state":"HI","rate":49.3},
+        {"state":"IA","rate":10.1},
+        {"state":"ID","rate":13.1},
+        {"state":"IL","rate":10.2},
+        {"state":"IN","rate":9.1},
+        {"state":"KS","rate":9.6},
+        {"state":"KY","rate":11.6},
+        {"state":"LA","rate":10.0},
+        {"state":"MA","rate":31.7},
+        {"state":"MD","rate":13.3},
+        {"state":"ME","rate":20.5},
+        {"state":"MI","rate":12.4},
+        {"state":"MN","rate":15.5},
+        {"state":"MO","rate":12.0},
+        {"state":"MS","rate":7.4},
+        {"state":"MT","rate":17.2},
+        {"state":"NC","rate":11.7},
+        {"state":"ND","rate":17.4},
+        {"state":"NE","rate":16.2},
+        {"state":"NH","rate":10.4},
+        {"state":"NJ","rate":13.1},
+        {"state":"NM","rate":13.2},
+        {"state":"NV","rate":37.8},
+        {"state":"NY","rate":41.0},
+        {"state":"OH","rate":10.2},
+        {"state":"OK","rate":10.9},
+        {"state":"OR","rate":31.0},
+        {"state":"PA","rate":12.0},
+        {"state":"PR","rate":11.4},
+        {"state":"RI","rate":11.3},
+        {"state":"SC","rate":10.6},
+        {"state":"SD","rate":10.5},
+        {"state":"TN","rate":14.5},
+        {"state":"TX","rate":10.8},
+        {"state":"UT","rate":10.6},
+        {"state":"VA","rate":8.5},
+        {"state":"VT","rate":24.9},
+        {"state":"WA","rate":26.5},
+        {"state":"WI","rate":10.5},
+        {"state":"WV","rate":10.9},
+        {"state":"WY","rate":13.0}
+      ]; 
+
+      var MAX_RATE = 120; 
```

Comment:
Ah, the DC problem. To include it or not to include it. (This is not in response to your approach, just the comment equivalent of a sigh.)